activerecord/lib/active_record/connection_adapters/abstract_adapter.rb

```diff
@@ -174,6 +175,14 @@ def schema_migration # :nodoc:
                               end
       end
 
+      def prepared_statements # :nodoc:
```

Comment:
Should this be documented? Sounds like the attribute was before.

Reply:
It was previously an `attr_reader` without documentation.

Reply:
Right, so it's listed under Attributes in https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/AbstractAdapter.html. `:nodoc:` means "this is explicitly undocumented/unsupported, and not part of our promised API". To be clear, I'm not suggesting we need to write words.. just that it shouldn't disappear completely.

```suggestion
      def prepared_statements
```

Reply:
Ah! Ok, got it.